packages/orchestrator-ui-components/src/messages/en-GB.json

```
@@ -47,6 +47,7 @@
             "cancel": "Cancel",
             "submit": "Submit",
             "previous": "Previous",
+            "previousQuestion": "Are you sure to leave this page? Any data filled on this page will be lost",
```

I think the below text is a bit cleaner
```suggestion
            "previousQuestion": "Are you sure you want to leave this page? Any filled form data will be lost",
```